prep-javascript-objects/main.js

```diff
@@ -0,0 +1,9 @@
+var person = { Firstname : "Walter", LastName: "White", Hobby: "Meth"}
+console.log(person)
+var fullName = "Walter White"
+console.log(fullName)
```

Comment:
We can constructor the person's full name by combining their `firstName` and `lastName` properties.

```js
var fullName = person.firstName + ' ' + person.lastName;
```